vignettes/articles/usethis-setup.Rmd

```diff
@@ -55,7 +55,7 @@ if (interactive()) {
 More resources on `.Rprofile`:
 
   * The official [Startup documentation](https://stat.ethz.ch/R-manual/R-patched/library/base/html/Startup.html)
-  * The [Startup chapter](https://whattheyforgot.org/r-startup.html) of the What They Forgot to Teach You About R workshop materials
+  * The [Startup chapter](https://rstats.wtf/) of the What They Forgot to Teach You About R workshop materials
```

Comment:
Yeah I'm learning the hard way that the original domain, which I let lapse, has been taken over for something else.

Can you update the fix to link to the specific page or section as the original intention? Or let me know if you can't determine what the original intent was.

Reply:
I just realized I could have replied here lol. I'm new to creating issues. Thank you for the link to the tidy code review guide.